lib/slack/web/docs/channels.archive.json

```diff
@@ -1,17 +0,0 @@
-{
```

Comment:
ℹ️  Deprecated. Do we want to keep these files and mark them as deprecated instead? Can we do that using meta programming?